website/docs/API.md

```diff
@@ -669,3 +680,23 @@ it('should use context value', () => {
   // ...
 });
 ```
+
+## Accessibility
+
+### `isInaccessible`
+
+```ts
+function isInaccessible(
+  element: ReactTestInstance | null
+): boolean {}
+```
+
+Checks if given element should be excluded from the accessiblity e.g. by screen readers. 
```

Comment:
"from the accessiblity" what? "from the accessibility checks" maybe?

Reply:
The text is based on DTLs "This function will compute if the given element should be excluded from the accessibility API by the browser. It implements every MUST criteria from the [Excluding Elements from the Accessibility Tree](https://www.w3.org/TR/wai-aria-1.2/#tree_exclusion) section in WAI-ARIA 1.2 with the exception of checking the role attribute."

However the first part of the original text is confusing, as presentational elements are excluded from accessibility API but the function will say that they aren't excluded. I'll update the description to be more accurate.

Reply:
I've updated the doc, and explained the behavior of the functions and it's relation to DTL's counterpart and ARIA in the simpliest terms I could :-)